Include/internal/pycore_condvar.h

```diff
@@ -9,7 +9,7 @@
 
 
 #ifdef _POSIX_THREADS
-/*
+/*ru
```

Comment:
```suggestion
/*
```
Was this accidentally added, what does it refer to otherwise?